owncloudApp/build.gradle

```diff
@@ -3,48 +3,43 @@ apply plugin: 'com.android.application'
 def commitSHA1 = 'COMMIT_SHA1'
 def gitRemote = 'GIT_REMOTE'
 
-configurations.all {
-    resolutionStrategy {
-        force "com.android.support:support-annotations:${supportLibraryVersion}"
-    }
-}
-
 dependencies {
     /// dependencies for app building
     implementation project(':owncloud-android-library:owncloudComLibrary')
-    implementation "com.android.support:support-v4:${supportLibraryVersion}"
-    implementation "com.android.support:design:${supportLibraryVersion}"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "androidx.legacy:legacy-support-v4:$androidX"
+    implementation 'com.google.android.material:material:1.0.0'
     implementation 'com.jakewharton:disklrucache:2.0.2'
     implementation 'com.google.android.exoplayer:exoplayer:r2.2.0'
     implementation 'com.andrognito.patternlockview:patternlockview:1.0.0'
-    implementation "com.android.support:appcompat-v7:${supportLibraryVersion}"
+    implementation 'androidx.appcompat:appcompat:1.0.0'
```

Comment:
Why don't you get the version from `$androidX` here?

Reply:
because 
`androidX` is `1.0.0` and 
`androidx.appcompat:appcompat` has currently `1.0.2`
But my fault I forgot to apply `1.0.2` here

done